docs/user/index.rst

```diff
@@ -1,11 +1,9 @@
 User Guide
 ==========
-.. note::
-  This information applies to end-users, not developers. We may want to find a more appropriate location for end-user information.
-
 
 .. toctree::
    :maxdepth: 1
 
-   installation
-   cli
+   learner
+   coach
+   admin
```

Comment:
Maybe put 'Admin' first? (These will be the users most likely to look at the docs and who need to set up initially)